Exiled.API/Extensions/RoleExtensions.cs

```diff
@@ -28,21 +28,21 @@ namespace Exiled.API.Extensions
     public static class RoleExtensions
     {
         /// <summary>
-        /// Gets a <see cref="RoleTypeId">role's</see> <see cref="Color"/>.
+        /// Gets <see cref="RoleTypeId">role's</see> <see cref="Color"/>.
```

Comment:
```suggestion
        /// Gets a <see cref="RoleTypeId">role's</see> <see cref="Color"/>.
```
The "a" is appropriate here - "Gets role's color" is not grammatically valid

Reply:
Reverted